.github/workflows/on_pr.yaml

```diff
@@ -0,0 +1,46 @@
+name: On Pull Request
+
+on: pull_request
+
+jobs:
+  build:
+    name: Build projects
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile
+
+      - name: Build api
+        run: cd packages/api && yarn build
+
+      - name: Build front-end
+        run: cd packages/front && yarn build
+
+      - name: Build store
+        run: cd packages/store && yarn build
+
+  unit-tests-store:
```

Comment:
Acho que podemos alterar a organização dos jobs para futuramente ter os 3: store-ci, front-ci, back-ci. Por enquanto vamos ficar apenas com o store-ci mesmo, e aí nele a gente pode rodar o yarn install, yarn build e o yarn test.

Dessa forma fica mais claro quando algo quebrar qual é exatamente o package, além de que se for necessário utilizar os arquivos de build para algum processamento ou link com o Heroku (quero testar isso depois!!) conseguimos obter direto desse job.